### ISSUE TITLE
mojmaps causes a mixin crash on fabric server launch

### ISSUE THREAD

**Issue by Terrails** [architectury/architectury-loom#51]
Like previously stated in the discord server the issue doesn't happen while using yarn or if using mojmaps/parchment inside a 'vanilla' fabric project. The issue was also happening on 0.7.4 and is still happening on 0.10.

Server log:
```[Architectury Transformer] Architectury Runtime 5.1.55
[Architectury Transformer] Read classpath in 4.379 s
[Architectury Transformer] Listening at D:\Programming\Minecraft\StatsKeeper\common\build\libs\statskeeper-1.17.1-5.0.0-dev.jar
[17:16:37] [main/INFO] (FabricLoader) Loading for game Minecraft 1.17.1
[17:16:37] [main/INFO] (Fabric|Loader) [FabricLoader] Loading 53 mods:
	- fabric@0.40.1+1.17
	- fabric-api-base@0.3.0+a02b446318
	- fabric-api-lookup-api-v1@1.3.0+cbda931818
	- fabric-biome-api-v1@3.2.0+cbda931818
	- fabric-blockrenderlayer-v1@1.1.5+a02b446318
	- fabric-command-api-v1@1.1.3+5ab9934c18
	- fabric-commands-v0@0.2.2+92519afa18
	- fabric-containers-v0@0.1.12+cbda931818
	- fabric-content-registries-v0@0.3.0+cbda931818
	- fabric-crash-report-info-v1@0.1.5+be9da31018
	- fabric-dimensions-v1@2.0.11+6cefd57718
	- fabric-entity-events-v1@1.2.3+87cc6e4c18
	- fabric-events-interaction-v0@0.4.10+fc40aa9d18
	- fabric-events-lifecycle-v0@0.2.1+92519afa18
	- fabric-game-rule-api-v1@1.0.7+cbda931818
	- fabric-gametest-api-v1@1.0.0+e409d7e818
	- fabric-item-api-v1@1.2.4+cbda931818
	- fabric-item-groups-v0@0.2.10+b7ab612118
	- fabric-key-binding-api-v1@1.0.4+cbda931818
	- fabric-keybindings-v0@0.2.2+36b77c3e18
	- fabric-lifecycle-events-v1@1.4.4+a02b446318
	- fabric-loot-tables-v1@1.0.4+a02b446318
	- fabric-mining-levels-v0@0.1.3+92519afa18
	- fabric-models-v0@0.3.0+a02b446318
	- fabric-networking-api-v1@1.0.13+cbda931818
	- fabric-networking-blockentity-v0@0.2.11+a02b446318
	- fabric-networking-v0@0.3.2+92519afa18
	- fabric-object-builder-api-v1@1.10.9+cbda931818
	- fabric-object-builders-v0@0.7.3+a02b446318
	- fabric-particles-v1@0.2.4+a02b446318
	- fabric-registry-sync-v0@0.7.10+e2961fee18
	- fabric-renderer-api-v1@0.4.4+cbda931818
	- fabric-renderer-indigo@0.4.8+cbda931818
	- fabric-renderer-registries-v1@3.2.3+4658223018
	- fabric-rendering-data-attachment-v1@0.1.5+a02b446318
	- fabric-rendering-fluids-v1@0.1.14+4658223018
	- fabric-rendering-v0@1.1.4+4658223018
	- fabric-rendering-v1@1.8.2+ffb6d41e18
	- fabric-resource-loader-v0@0.4.8+a00e834b18
	- fabric-screen-api-v1@1.0.4+198a96213d
	- fabric-screen-handler-api-v1@1.1.8+cbda931818
	- fabric-structure-api-v1@1.1.13+5ab9934c18
	- fabric-tag-extensions-v0@1.2.1+b06cb95b18
	- fabric-textures-v0@1.0.6+a02b446318
	- fabric-tool-attribute-api-v1@1.2.12+b7ab612118
	- fabric-transfer-api-v1@1.3.1+4658223018
	- fabricloader@0.11.7
	- fiber@0.23.0-2
	- generated_809dd26@1.0.0
	- java@16
	- minecraft@1.17.1
	- modmenu@2.0.12
	- statskeeper@5.0.0
[17:16:38] [main/INFO] (mixin) SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/C:/Users/Filip/.gradle/caches/modules-2/files-2.1/net.fabricmc/sponge-mixin/0.9.4+mixin.0.8.2/ef410951e94c7f4939cc74c86f8df371827818a4/sponge-mixin-0.9.4+mixin.0.8.2.jar Service=Knot/Fabric Env=SERVER
[17:16:38] [main/INFO] (Fabric|MixinBootstrap) Loaded Fabric development mappings for mixin remapper!
[17:16:38] [main/INFO] (mixin) Compatibility level set to JAVA_16
[17:16:38] [main/WARN] (mixin) @Mixin target net/minecraft/server/level/ServerLevel$EntityCallbacks is public in fabric-lifecycle-events-v1.mixins.json:ServerWorldEntityLoaderMixin and should be specified in value
[17:16:38] [main/WARN] (mixin) @Mixin target net/minecraft/server/network/ServerGamePacketListenerImpl$1 is public in fabric-events-interaction-v0.mixins.json:MixinServerPlayNetworkHandler and should be specified in value
[17:16:38] [main/WARN] (mixin) @Mixin target net/minecraft/world/entity/npc/VillagerTrades$EmeraldsForVillagerTypeItem is public in fabric-object-builder-v1.mixins.json:TypeAwareTradeMixin and should be specified in value
[17:16:38] [main/WARN] (mixin) @Mixin target net/minecraft/server/commands/GameRuleCommand$1 is public in fabric-game-rule-api-v1.mixins.json:GameRuleCommandVisitorMixin and should be specified in value
[17:16:38] [main/WARN] (mixin) @Mixin target net/minecraft/server/level/ChunkMap$TrackedEntity is public in fabric-networking-api-v1.mixins.json:accessor.EntityTrackerAccessor and should be specified in value
[17:16:38] [main/WARN] (mixin) @Mixin target net/minecraft/data/tags/TagsProvider$TagAppender is public in fabric-tag-extensions-v0.mixins.json:MixinObjectBuilder and should be specified in value
[17:16:38] [main/WARN] (mixin) @Mixin target net/minecraft/tags/StaticTagHelper$Wrapper is public in fabric-tag-extensions-v0.mixins.json:MixinTagImpl and should be specified in value
[17:16:39] [main/WARN] (mixin) Error loading class: net/minecraft/world/level/storage/LevelStorage$Session (java.lang.ClassNotFoundException: net/minecraft/world/level/storage/LevelStorage$Session)
[17:16:39] [main/FATAL] (mixin) Mixin apply failed fabric-gametest-api-v1.mixins.json:server.MainMixin -> net.minecraft.server.Main: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Error whilst instancing injection point org.spongepowered.asm.mixin.injection.points.BeforeInvoke for INVOKE [PREINJECT Applicator Phase -> fabric-gametest-api-v1.mixins.json:server.MainMixin -> Prepare Injections ->  -> handler$zzn000$main([Ljava/lang/String;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Ljoptsimple/OptionParser;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSet;Lnet/minecraft/core/RegistryAccess$RegistryHolder;Ljava/nio/file/Path;Lnet/minecraft/server/dedicated/DedicatedServerSettings;Ljava/nio/file/Path;Lnet/minecraft/server/Eula;Ljava/io/File;Lcom/mojang/authlib/yggdrasil/YggdrasilAuthenticationService;Lcom/mojang/authlib/minecraft/MinecraftSessionService;Lcom/mojang/authlib/GameProfileRepository;Lnet/minecraft/server/players/GameProfileCache;Ljava/lang/String;Lnet/minecraft/world/level/storage/LevelStorageSource;Lnet/minecraft/world/level/storage/LevelStorageSource$LevelStorageAccess;Lnet/minecraft/world/level/storage/LevelSummary;Lnet/minecraft/world/level/DataPackConfig;ZLnet/minecraft/server/packs/repository/PackRepository;Lnet/minecraft/world/level/DataPackConfig;Ljava/util/concurrent/CompletableFuture;Lnet/minecraft/server/ServerResources;Lnet/minecraft/resources/RegistryReadOps;)V -> Parse]
 org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Error whilst instancing injection point org.spongepowered.asm.mixin.injection.points.BeforeInvoke for INVOKE [PREINJECT Applicator Phase -> fabric-gametest-api-v1.mixins.json:server.MainMixin -> Prepare Injections ->  -> handler$zzn000$main([Ljava/lang/String;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Ljoptsimple/OptionParser;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSet;Lnet/minecraft/core/RegistryAccess$RegistryHolder;Ljava/nio/file/Path;Lnet/minecraft/server/dedicated/DedicatedServerSettings;Ljava/nio/file/Path;Lnet/minecraft/server/Eula;Ljava/io/File;Lcom/mojang/authlib/yggdrasil/YggdrasilAuthenticationService;Lcom/mojang/authlib/minecraft/MinecraftSessionService;Lcom/mojang/authlib/GameProfileRepository;Lnet/minecraft/server/players/GameProfileCache;Ljava/lang/String;Lnet/minecraft/world/level/storage/LevelStorageSource;Lnet/minecraft/world/level/storage/LevelStorageSource$LevelStorageAccess;Lnet/minecraft/world/level/storage/LevelSummary;Lnet/minecraft/world/level/DataPackConfig;ZLnet/minecraft/server/packs/repository/PackRepository;Lnet/minecraft/world/level/DataPackConfig;Ljava/util/concurrent/CompletableFuture;Lnet/minecraft/server/ServerResources;Lnet/minecraft/resources/RegistryReadOps;)V -> Parse]
	at org.spongepowered.asm.mixin.injection.InjectionPoint.create(InjectionPoint.java:734) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.InjectionPoint.parse(InjectionPoint.java:698) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.InjectionPoint.parse(InjectionPoint.java:648) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.InjectionPoint.parse(InjectionPoint.java:569) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parseInjectionPoints(InjectionInfo.java:330) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.readAnnotation(InjectionInfo.java:289) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:275) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:267) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.CallbackInjectionInfo.<init>(CallbackInjectionInfo.java:46) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo$InjectorEntry.create(InjectionInfo.java:140) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parse(InjectionInfo.java:624) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1247) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1033) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:388) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) [sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) [sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) [sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) [sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) [fabric-loader-0.11.7.jar:0.9.4+mixin.0.8.2]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:162) [fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:154) [fabric-loader-0.11.7.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:519) [?:?]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:232) [fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153) [fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:28) [fabric-loader-0.11.7.jar:?]
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) [dev-launch-injector-0.2.1+build.8.jar:?]
	at dev.architectury.transformer.TransformerRuntime.main(TransformerRuntime.java:297) [architectury-transformer-5.1.55-runtime.jar:5.1.55]
Caused by: java.lang.reflect.InvocationTargetException
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at org.spongepowered.asm.mixin.injection.InjectionPoint.create(InjectionPoint.java:732) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	... 33 more
Caused by: java.lang.NullPointerException
	at java.util.ArrayDeque.addLast(ArrayDeque.java:303) ~[?:?]
	at java.util.ArrayDeque.add(ArrayDeque.java:494) ~[?:?]
	at net.fabricmc.loader.util.mappings.MixinIntermediaryDevRemapper.mapMethodName(MixinIntermediaryDevRemapper.java:142) ~[fabric-loader-0.11.7.jar:?]
	at org.spongepowered.asm.obfuscation.RemapperChain.mapMethodName(RemapperChain.java:59) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.refmap.RemappingReferenceMapper.remapWithContext(RemappingReferenceMapper.java:164) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.refmap.RemappingReferenceMapper.remap(RemappingReferenceMapper.java:121) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.MemberInfo.parse(MemberInfo.java:692) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.selectors.TargetSelector.parse(TargetSelector.java:154) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.selectors.TargetSelector.parseAndValidate(TargetSelector.java:109) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionPointData.getTarget(InjectionPointData.java:273) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.points.BeforeInvoke.<init>(BeforeInvoke.java:126) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at org.spongepowered.asm.mixin.injection.InjectionPoint.create(InjectionPoint.java:732) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	... 33 more
Exception in thread "main" org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178)
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23)
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:162)
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:154)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:519)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:232)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153)
	at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:28)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)
	at dev.architectury.transformer.TransformerRuntime.main(TransformerRuntime.java:297)
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [fabric-gametest-api-v1.mixins.json:server.MainMixin] from phase [DEFAULT] in config [fabric-gametest-api-v1.mixins.json] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:642)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:594)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:356)
	... 11 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Error whilst instancing injection point org.spongepowered.asm.mixin.injection.points.BeforeInvoke for INVOKE [PREINJECT Applicator Phase -> fabric-gametest-api-v1.mixins.json:server.MainMixin -> Prepare Injections ->  -> handler$zzn000$main([Ljava/lang/String;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Ljoptsimple/OptionParser;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSpec;Ljoptsimple/OptionSet;Lnet/minecraft/core/RegistryAccess$RegistryHolder;Ljava/nio/file/Path;Lnet/minecraft/server/dedicated/DedicatedServerSettings;Ljava/nio/file/Path;Lnet/minecraft/server/Eula;Ljava/io/File;Lcom/mojang/authlib/yggdrasil/YggdrasilAuthenticationService;Lcom/mojang/authlib/minecraft/MinecraftSessionService;Lcom/mojang/authlib/GameProfileRepository;Lnet/minecraft/server/players/GameProfileCache;Ljava/lang/String;Lnet/minecraft/world/level/storage/LevelStorageSource;Lnet/minecraft/world/level/storage/LevelStorageSource$LevelStorageAccess;Lnet/minecraft/world/level/storage/LevelSummary;Lnet/minecraft/world/level/DataPackConfig;ZLnet/minecraft/server/packs/repository/PackRepository;Lnet/minecraft/world/level/DataPackConfig;Ljava/util/concurrent/CompletableFuture;Lnet/minecraft/server/ServerResources;Lnet/minecraft/resources/RegistryReadOps;)V -> Parse]
	at org.spongepowered.asm.mixin.injection.InjectionPoint.create(InjectionPoint.java:734)
	at org.spongepowered.asm.mixin.injection.InjectionPoint.parse(InjectionPoint.java:698)
	at org.spongepowered.asm.mixin.injection.InjectionPoint.parse(InjectionPoint.java:648)
	at org.spongepowered.asm.mixin.injection.InjectionPoint.parse(InjectionPoint.java:569)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parseInjectionPoints(InjectionInfo.java:330)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.readAnnotation(InjectionInfo.java:289)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:275)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:267)
	at org.spongepowered.asm.mixin.injection.struct.CallbackInjectionInfo.<init>(CallbackInjectionInfo.java:46)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo$InjectorEntry.create(InjectionInfo.java:140)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parse(InjectionInfo.java:624)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1247)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1033)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:388)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351)
	... 11 more
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.spongepowered.asm.mixin.injection.InjectionPoint.create(InjectionPoint.java:732)
	... 33 more
Caused by: java.lang.NullPointerException
	at java.base/java.util.ArrayDeque.addLast(ArrayDeque.java:303)
	at java.base/java.util.ArrayDeque.add(ArrayDeque.java:494)
	at net.fabricmc.loader.util.mappings.MixinIntermediaryDevRemapper.mapMethodName(MixinIntermediaryDevRemapper.java:142)
	at org.spongepowered.asm.obfuscation.RemapperChain.mapMethodName(RemapperChain.java:59)
	at org.spongepowered.asm.mixin.refmap.RemappingReferenceMapper.remapWithContext(RemappingReferenceMapper.java:164)
	at org.spongepowered.asm.mixin.refmap.RemappingReferenceMapper.remap(RemappingReferenceMapper.java:121)
	at org.spongepowered.asm.mixin.injection.struct.MemberInfo.parse(MemberInfo.java:692)
	at org.spongepowered.asm.mixin.injection.selectors.TargetSelector.parse(TargetSelector.java:154)
	at org.spongepowered.asm.mixin.injection.selectors.TargetSelector.parseAndValidate(TargetSelector.java:109)
	at org.spongepowered.asm.mixin.injection.struct.InjectionPointData.getTarget(InjectionPointData.java:273)
	at org.spongepowered.asm.mixin.injection.points.BeforeInvoke.<init>(BeforeInvoke.java:126)
	... 39 more
```

**Comment by kirderf1** [comment]
Looks like this issue was just fixed with fabric api 0.40.8.

**Comment by shedaniel** [comment]
Thanks for confirming
